code/index.html

```diff
@@ -1,22 +1,84 @@
 <!DOCTYPE html>
 <html>
+
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <title>Knitty&#9829News</title>
     <link rel="stylesheet" href="./style.css" />
     <!-- dont forget the meta tag for responsiveness -->
+    <meta name="viewport" content="width=device-width, initial-scale: 1.0">
+    <!--Google fonts-->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Yusei+Magic&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <header>
+    <header class="header">
       <!-- Some kind of logo and navbar -->
+      <div class="flex-container">
+        
+        <div class="logo">    
+          <img src="./KN_logo.svg" alt="logotype KnittyNews" />
+        </div>
+
+        <div class="nav">
+          <div class="nav-item"><a href="#">About</a></div>
+          <div class="nav-item"><a href="#">Archieve</a></div>
+          <div class="nav-item"><a href="#">Forum</a></div>
+          <div class="nav-item"><a href="#">Local</a></div>
+          <div class="nav-item"><a href="#">Contact</a></div>
```

Comment:
Do you need these wrapping `div`s or would it be enough with just the links? If you decide to keep, give nested elements new lines:
```
<div>
  <a></a>
</div>
```